ceph_iscsi_config/client.py

```diff
@@ -234,6 +234,30 @@ def define_client(self):
                              "successfully".format(self.iqn))
             self.change_count += 1
 
+    @staticmethod
+    def get_client_info(target_iqn, client_iqn):
+        result = {
+            "alias": '',
+            "state": '',
+            "ip_address": []
+        }
+        iscsi_fabric = ISCSIFabricModule()
+        target = Target(iscsi_fabric, target_iqn, 'lookup')
+        for tpg in target.tpgs:
+            if tpg.enable:
+                for client in tpg.node_acls:
+                    if client.session and client.session['parent_nodeacl'].node_wwn == client_iqn:
```

Comment:
I'm not sure I got the parent_nodeacl part. We know the session is through 'client' so we can just access the client.node_wwn. session['parent_nodeacl'] will always be the client we access the session through, so we can just do:

if client.node_wwn != client_iqn:
    continue;

# temp comment for review only. There can only be one client with client_iqn under each tpg, so once we find it, we know the search is done and we can break from the "for client in tpg.node_acls:" loop.

session = client.session:
if session is None:
    break

# copy session info to result
break